src/org/rascalmpl/interpreter/utils/ReadEvalPrintDialogMessages.java

```diff
@@ -206,4 +206,9 @@ public static void staticErrorMessage(PrintWriter out, StaticError e, StandardTe
 	    out.println(e.getMessage());
 	    out.flush();
 	}
+
+	public static void cancelledMessage(PrintWriter out) {
+		out.println("Cancelled");
```

Comment:
Just like `interrupted` and other messages, they can be inligned at the single place where they're printed. This class was made for larger messages. (and it used to be shared between a few classes)